app/graphql/schema.graphql

```diff
@@ -8418,6 +8420,166 @@ type Query {
   works(after: String, facetCount: Int = 10, fieldOfScience: String, first: Int = 25, hasAffiliation: Boolean, hasCitations: Int, hasDownloads: Int, hasFunder: Boolean, hasMember: Boolean, hasOrganization: Boolean, hasParts: Int, hasPerson: Boolean, hasVersions: Int, hasViews: Int, ids: [String!], language: String, license: String, memberId: String, published: String, query: String, registrationAgency: String, repositoryId: String, resourceType: String, resourceTypeId: String, userId: String): WorkConnectionWithTotal!
 }
 
+"""
+Information about repositories
+"""
+type ReferenceRepository {
+  """
+  Repository alternate names
+  """
+  alternateName: [String!]
+
+  """
+  The certificate(s) for the repository
+  """
+  certificate: [String!]
+
+  """
+  The number of citations.
+  """
+  citationCount: Int
+
+  """
+  The unique identifier for the client repository
+  """
+  clientId: ID
+
+  """
+  Repository contact information
+  """
+  contact: [String!]
+
+  """
+  Data accesses
+  """
+  dataAccess: [TextRestriction!]
+
+  """
+  Data uploads
+  """
+  dataUpload: [TextRestriction!]
+
+  """
+  Description of the repository
+  """
+  description: String
+
+  """
+  The number of downloads according to the Counter Code of Practice.
+  """
+  downloadCount: Int
+
+  """
+  The langauge of the repository
+  """
+  language: [String!]
+
+  """
+  Repository name
+  """
+  name: String!
+
+  """
+  PID Systems
+  """
+  pidSystem: [String!]
```

Comment:
The r3d schema has a `minOccurs` of 0 for this property - should it definitely be non-null?